config/source.json

```diff
@@ -388,6 +388,15 @@
             "path": "LICENSE"
         }
     },
+    "jbig": {
+        "type": "url",
+        "url": "https://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-2.1.tar.gz",
```

Comment:
How about using https://github.com/ImageMagick/jbig or self-hosted here and make this as alt? I cannot pull source from cambridge with stable speed, and it has not been updated in 11 years.